### ISSUE TITLE
Default list of scanner Sane backends all disabled

### ISSUE THREAD

**Issue by xlwyx** [geckolinux/geckolinux-project#213]
It is not possible that the mentioned applications will not localize the scanner device.
I used HP-scanjet-G2710 & Epson Scan-L475 (It's kind of outrageous.)...

I'm about to blacklist geckolinux tw

**Comment by xlwyx** [comment]
> It is not possible that the mentioned applications will not localize the scanner device.
> I used HP-scanjet-G2710 & Epson Scan-L475 (It's kind of outrageous.)...
> 
> I'm about to blacklist geckolinux tw

https://i.imgur.com/ViXobuz.png
https://i.imgur.com/BenqLuf.png
https://i.imgur.com/mpk6gRY.png
https://i.imgur.com/gikAELP.png
https://i.imgur.com/6V4rzAc.png
https://i.imgur.com/Fcdev8d.png
https://i.imgur.com/Oht1wSN.png
https://i.imgur.com/Vj688D1.png
https://i.imgur.com/zfRBaQ7.png

**Comment by geckolinux** [comment]
You need to configure the drivers, all frontend applications will fail if the scanner isn't properly configured to work with SANE.

For the Epson, can't be preinstalled with GeckoLinux, download and follow the README instructions:
https://download2.ebz.epson.net/imagescanv3/opensuse/latest1/rpm/x64/imagescan-bundle-opensuse-15.2-3.65.0.x64.rpm.tar.gz

The necessary SANE backend for the HP is already installed with GeckoLinux:
http://www.sane-project.org/man/sane-hp3900.5.html

**Comment by gioKheper** [comment]
Hi, the problem seems to be here again. I tried to use Hp Scanjet 3880, that is working with, for instance, OpenSuse Leap 15.3, but the Sane does not find any scanner at all. The same with a pc where I have installed  OpenSuse Tumbleweed. By the way my version of Gecko is rolling Plasma.

**Comment by geckolinux** [comment]
Hi @gioKheper , please make sure that the `sane-backends` package is still installed. It is pre-installed in GeckoLinux and includes the sane-hp3900 backend needed for your printer.
http://www.sane-project.org/man/sane-hp3900.5.html

**Comment by gioKheper** [comment]
Hi, thank you. For what I can see the file hp3900.conf is there under /etc/sane.d directory and is configured as it should be. I tried to reinstall  sane-backends, now at version 1.1.1-1.4, both on tumbleweed and on gecko, but with the same results, i.e. skanlite cannot find the scanner and ask to reload the list of scanners.
I do not know if I am missing something else.

**Comment by geckolinux** [comment]
Could you also please check if `sane-backends-autoconfig` is installed?

**Comment by gioKheper** [comment]
Yes, it is installed both on Gecko and on Tumbleweed

**Comment by geckolinux** [comment]
OK, thanks for confirming that. Could you please post your `/etc/sane.d/dll.conf` from GeckoLinux and from a working vanilla openSUSE installation?

**Comment by gioKheper** [comment]
[dll.conf.zip](https://github.com/geckolinux/geckolinux-project/files/8760297/dll.conf.zip)
This is the one from the working OpenSuse 15.3

**Comment by gioKheper** [comment]
[dll.conf.zip](https://github.com/geckolinux/geckolinux-project/files/8760416/dll.conf.zip)
Thank you again.
Hi this is the one from Gecko. I noticed that in this one the line hp3900 is commented, so I tried commenting it out and now it is working.
I do not know why the dll.conf are different because I did not touch them, but now it is working. 
By the way my scanner is not connected via network, but just via an USB port.

**Comment by geckolinux** [comment]
Glad you figured it out. It looks like the `dll.conf` that openSUSE ships (and thus what GeckoLinux uses) has all the backends commented by default. I wonder if you used the YaST Scanner tool under Leap 15.3 to automatically enable the hp3900 backend. But thanks for bringing this up, it looks like things could be optimized a bit for out-of-the-box scanner functionality in GeckoLinux.

**Comment by gioKheper** [comment]
Well, I am not sure I used Yast.scanner for openSuse 15.3. I tried to use it with Tumbleweed and I got 2 strange things:

1. I plugged in the Hp3800, started yast.scanner: the program showed the scanner but not configured so I was not able to test it.The file dll.conf was untouched. After I commented out the proper line hp3900, the program showed me the scanner as configured and I was able to test it.
2. I plugged in another scanner I have, an HP5300C recognized as avision. This was properly recognized as avision and configured, and the dll.conf was changed. By the way I was not able make it running because I got an I/O error (This is an error I get for this scanner on any linux I use now, so not sure the problem is a now broken scanner that was working fine with very old linuxes many years ago or a problem with the avision backend, I need to investigate,but not now)
